ng2-components/ng2-alfresco-viewer/src/componets/viewer.component.html

```diff
@@ -49,23 +49,27 @@
                          class="center-element mdl-cell mdl-cell--12-col">
 
                         <!-- Start View Switch-->
-                        <div *ngIf="isPdf()">
-                            <pdf-viewer [showToolbar]="showToolbar" [urlFile]="urlFileContent"
-                                        [nameFile]="displayName"></pdf-viewer>
-                        </div>
-                        <div class="center-element" *ngIf="isImage()">
-                            <img-viewer [urlFile]="urlFileContent" [nameFile]="displayName"></img-viewer>
-                        </div>
-                        <div class="center-element" *ngIf="isMedia()">
-                            <media-player [urlFile]="urlFileContent" [mimeType]="mimeType"
-                                          [nameFile]="displayName"></media-player>
-                        </div>
-                        <div *ngIf="!supportedExtension()">
-                            <not-supported-format [urlFile]="urlFileContent"
-                                                  [nameFile]="displayName"></not-supported-format>
-                        </div>
+                        <!-- <div *ngIf="isPdf()"> -->
```

Comment:
we can not remove existing functionality, a better idea could be put there a ngIf, and in case a iframeurl is defined, display the iframe.